submission/proxy_server.go

```diff
@@ -44,6 +44,7 @@ func NewProxyServer(logListPath string, dBuilder DistributorBuilder, reqTimeout
 // Run starts regular Log list updates.
```

Comment:
Should document that this will block until initialisation is successful, which could be indefinitely if there's a problem.

Reply:
Done